artemis-cli/src/test/java/org/apache/activemq/cli/test/CliProducerTest.java

```diff
@@ -54,9 +54,11 @@ public void tearDown() throws Exception {
       super.tearDown();
    }
 
-   private void produceMessages(String address, String message, int msgCount) throws Exception {
-      new Producer()
+   private Object produceMessages(String address, String message, int msgCount, String properties) throws Exception {
+      System.out.println(properties);
```

Comment:
I wouldn't use System.out.println here.. would keep logger.debug

Reply:
I forgot to remove that.